guides/common/modules/ref_supported-operating-systems.adoc

```diff
@@ -28,17 +24,11 @@ ifdef::foreman-deb[]
 endif::[]
 |====
 
-Before you install {Project}, apply all operating system updates if possible.
-
+{Team} advises against using an existing system, since the installer will affect the configuration of several components.
```

Comment:
```suggestion
{Team} advises against using an existing system because the {Project} installer will affect the configuration of several components.
```

Reply:
ACK

Reply:
Done